src/sql/workbench/services/dialog/browser/customDialogService.ts

```diff
@@ -9,8 +9,8 @@ import { Dialog, Wizard } from 'sql/workbench/services/dialog/common/dialogTypes
 import { IModalOptions } from 'sql/workbench/browser/modal/modal';
 import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
 
-export const DefaultDialogOptions: IModalOptions = { hasBackButton: false, width: 'narrow', hasErrors: true };
-export const DefaultWizardOptions: IModalOptions = { hasBackButton: false, width: 'wide', hasErrors: true };
+export const DefaultDialogOptions: IModalOptions = { hasBackButton: false, width: 'narrow', hasErrors: true, hasSpinner: true };
```

Comment:
Just to make sure - the spinner isn't tabbable or cause any issues with screen readers if loading isn't set to true right?

Reply:
It isn't tabbable. This spinner is not a new component. We already use it in connection dialog. So I am assuming it does not have any accessibility issue.